docs/manuals/source/Appendix/DisasterRecoveryUsingBareos.rst

```diff
@@ -473,38 +471,183 @@ and add the Run Script definition to the job intended to backup the data for a R
+Bareos media. The call-up syntax is:
+
+::
+
+   bextract -c <SD configuration directory>  <drive name in sd.conf> <output directory>  -b <bootstrap file>
+
+``bextract`` now extracts the data specified in the ``<Bootstrap file>``
+to the ``<output file>`` using drive ``<drive name>``. ``bextract``
+consults the storage daemon configuration file to determine how to
+address the drive.
+
+For our example, that means the following call:
+
+::
+
+   root@bareos:~ #  bextract -c /etc/bareos -V E01001L4  Drive-1 /tmp  -b BackupCatalog.bsr
+   bextract: butil.c:287 Using device: "Drive-1" for reading.
+   15-Mär 12:07 bextract JobId 0: 3301 Issuing autochanger "loaded? drive 0" command.
+   15-Mär 12:07 bextract JobId 0: 3302 Autochanger "loaded? drive 0", result is Slot 1.
+   15-Mär 12:07 bextract JobId 0: Ready to read from volume "E01001L4" on device "Drive-1" (/dev/nst0).
+   15-Mär 12:07 bextract JobId 0: Forward spacing Volume "E01001L4" to file:block 1:0.
+   bextract JobId 0: -rw-------   1 bareos   bareos         87862 2025-03-15 11:02:42  /tmp/var/lib/bareos/bareos.sql
+   15-Mär 12:07 bextract JobId 0: End of Volume at file 2 on device "Drive-1" (/dev/nst0), Volume "E01001L4"
+   15-Mär 12:07 bextract JobId 0: End of all volumes.
+   1 files restored.
+
+
+We have successfully extracted the database dump to file
+:file:`/tmp/var/lib/bareos/bareos.sql`. We now want to import it to
+the Bareos database, so we use Bareos scripts to create the database,
+tables and access permissions:
+
+::
+
+   root@bareos:/usr/lib/bareos/scripts # ./create_bareos_database
+   Creating postgresql database
+   CREATE DATABASE
+   ALTER DATABASE
+   Creation of bareos database succeeded.
+   Database encoding OK
+
+   root@bareos:/usr/lib/bareos/scripts # make_bareos_tables
+   Making postgresql tables
+   [...]
+   INSERT 0 1
+   Creation of Bareos PostgreSQL tables succeeded.
 
--  After the Catalog is restored, it should be located in (:file:`./var/lib/bareos/bareos.sql`) relative to the restore location. After stopping the bareos-dir service, this file then needs to be loaded into the PostgreSQL database. (For example by running ``sudo -u bareos psql bareos bareos -f ./var/lib/bareos/bareos.sql`` or ```su postgres -c "psql bareos -f ./var/lib/bareos/bareos.sql``)
+   root@bareos:/usr/lib/bareos/scripts # grant_bareos_privileges
 
--  If you have successfully used a Bootstrap file, you should now be back up and running, if you do not have a Bootstrap file, continue with the suggestions below.
+We now import the database dump to the database:
 
--  Using bscan scan the last set of backup tapes into your catalog database.
+::
 
--  Start Bareos, and using the Console restore command, restore the last valid copy of the Bareos database and the Bareos configuration files.
+   root@bareos:~ #  psql bareos < /tmp/var/lib/bareos/bareos.sql
 
--  Move the database to the correct location.
+Finally you start the director. The system is now restored with all
```

Comment:
I would also suggest here to move the restored /tmp/etc/bareos back to it's original location, so the daemons can be started system wise.
If we don't specify that step, there will be people starting their restored catalog with the new fresh default bareos etc, and of course some configuration will be missing like previously defined jobs, clients, fileset, pools etc ...

```suggestion

We bring back the restored configuration to its initial location
::
cp -a /tmp/etc/bareos /etc/bareos

Finally you restart the services (director,storage,filedaemon).
::
systemctl restart bareos-director bareos-storage bareos-filedaemon

The system is now restored with all
```